test/TestServerProjectsLowLevel/dpg-customization/Generated/DPGClient.cs

```diff
@@ -0,0 +1,415 @@
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get models that you will either return to end users as a raw body, or with a model added during grow up. </summary>
+        /// <param name="mode"> The mode with which you&apos;ll be handling your returned body. &apos;raw&apos; for just dealing with the raw body, and &apos;model&apos; if you are going to convert the raw body to a customized body before returning to users. </param>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="mode"/> is null. </exception>
+        /// <remarks>
+        /// Schema for <c>Response Body</c>:
+        /// <code>{
+        ///   received: &quot;raw&quot; | &quot;model&quot;
+        /// }
+        /// </code>
+        /// 
+        /// </remarks>
+        public virtual Response GetModel(string mode, RequestContext context = null)
+        {
+            Argument.AssertNotNull(mode, nameof(mode));
+
+            using var scope = ClientDiagnostics.CreateScope("DPGClient.GetModel");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetModelRequest(mode, context);
+                return _pipeline.ProcessMessage(message, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Post either raw response as a model and pass in &apos;raw&apos; for mode, or grow up your operation to take a model instead, and put in &apos;model&apos; as mode. </summary>
+        /// <param name="mode"> The mode with which you&apos;ll be handling your returned body. &apos;raw&apos; for just dealing with the raw body, and &apos;model&apos; if you are going to convert the raw body to a customized body before returning to users. </param>
+        /// <param name="content"> The content to send as the body of the request. </param>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="mode"/> or <paramref name="content"/> is null. </exception>
+        /// <remarks>
+        /// Schema for <c>Request Body</c>:
+        /// <code>{
+        ///   hello: string (required)
+        /// }
+        /// </code>
+        /// Schema for <c>Response Body</c>:
+        /// <code>{
+        ///   received: &quot;raw&quot; | &quot;model&quot;
+        /// }
+        /// </code>
+        /// 
+        /// </remarks>
+        public virtual async Task<Response> PostModelAsync(string mode, RequestContent content, RequestContext context = null)
```

Comment:
If we are validating approach #2, this is correct.  If we decide on #4, we'll need to add a grow-up method that takes a model type as input and returns a model type as output.